attributes/default.rb

```diff
@@ -19,3 +19,12 @@
 #
 
 default['ms_dotnet']['timeout'] = 600
+
+if platform? 'windows'
+  nt_version = node['platform_version'].to_f
+  if nt_version >= 6.2
+    default['ms_dotnet']['feature']['enable_all']            = true
```

Comment:
You should place this attribute in the "v3" namespace i.e. `default['ms_dotnet']['v3']['feature']['enable_all']`